repose-aggregator/core/repose-core/src/main/java/org/openrepose/powerfilter/PowerFilter.java

```diff
@@ -424,8 +424,12 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             LOG.debug("{}:{} -- Invalid URI requested: {}", clusterId, nodeId, wrappedRequest.getRequestURI(), use);
             wrappedResponse.sendError(HttpServletResponse.SC_BAD_REQUEST, "Error processing request");
         } catch (Exception ex) {
-            LOG.error("{}:{} -- Exception encountered while processing filter chain.", clusterId, nodeId, ex);
+            LOG.error("{}:{} -- Issue encountered while processing filter chain.", clusterId, nodeId, ex);
             wrappedResponse.sendError(HttpServletResponse.SC_BAD_GATEWAY, "Error processing request");
```

Comment:
right here.

Reply:
Oh, right, but no. The exception actually gets caught before this, in the `PowerFilterChain`. I added a comment in the test saying as much. Here:

https://github.com/rackerlabs/repose/blob/master/repose-aggregator/core/repose-core/src/main/java/org/openrepose/powerfilter/PowerFilterChain.java#L218-L222

So I guess this block is for `Exception`s thrown outside of a filter.